continuous_integration/recipes/distributed/meta.yaml

```diff
@@ -42,7 +42,7 @@ requirements:
     - sortedcontainers !=2.0.0,!=2.0.1
     - tblib >=1.6.0
     - toolz >=0.8.2
-    - tornado >=6.0.3,<6.2
+    - tornado >=6.0.3
```

Comment:
Note that I just copied over the changes for #6991; in 3.10 pinning exactly to 6.2 ensures that we actually test with 6.2 in one CI setting. I don't know if the looser pins should all explicitly request `>= 6.0.3` (at present they pick up 6.1)